src/NuGet.Core/NuGet.Commands/RestoreCommand/LockFileBuilder.cs

```diff
@@ -231,7 +231,7 @@ public class LockFileBuilder
                         var package = packageInfo.Package;
                         var libraryDependency = tfi.Dependencies.FirstOrDefault(e => e.Name.Equals(library.Name, StringComparison.OrdinalIgnoreCase));
 
-                        var targetLibrary = LockFileUtils.CreateLockFileTargetLibrary(
+                        var (targetLibrary, usedFallbackFramework) = LockFileUtils.CreateLockFileTargetLibrary(
```

Comment:
I think this is a place where explicit types, rather than `var` is useful. I assumed that `usedFallbackFramework` was a `NuGetFramework`, or similar, so was surprised when it was used as a boolean. Now it makes sense, but in the spirit of reducing surprises and also our coding guidelines document which doesn't allow `var` here anyway, I think it should be changed since the line is being changed anyway.

Reply:
fixed